sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ActionDescriptorFactory.scala

```diff
@@ -183,11 +210,32 @@ private[impl] object ActionDescriptorFactory extends ComponentDescriptorFactory
         springMethods.exists(s => p.serviceMethod.methodName.equals(s.serviceMethod.methodName)))
     }
 
+    val serviceLevelOptions = {
```

Comment:
`typeLevelOptions` vielleicht?

Reply:
I'm failing to understand this change. How come we were only passing ` AclDescriptorFactory.serviceLevelAclAnnotation(component)` before but now all these options are needed?

Reply:
they all become gPRC service level annotations. 

Now, we can have ACL, but also eventing.in and eventing.out.direct at service level.